src/iris/ui/static/js/iris.js

```diff
@@ -2026,7 +2033,7 @@ iris = {
         app.editable = window.appData.user_admin || owner;
         app.supportViewingKey = window.appData.user_admin || owner;
         app.supportEditingEmailIncidents = window.appData.user_admin || owner;
-        app.showEditOwners = window.appData.user_admin;
+        app.showEditOwners = window.appData.user_admin || owner;
```

Comment:
I'm confused by 'owner' here. Wouldn't `var owner = app.owners.indexOf(window.appData.user) !== -1;` just return true/false? maybe a ternary statement here is what you were going for - var owner = app.owners.indexOf(window.appData.user) !== -1 ? app.owners[window.appData.user] : 'null';